pulumi/azure/managed-redis-managed-event-bus-temporal-clickhouse/launcher.ts

```diff
@@ -1,188 +1,467 @@
 import * as path from "path";
```

Comment:
Is there an opportunity to unify these launchers and have a repo wide launcher that coordinates everything?